src/core/lv2/Lv2Manager.cpp

```diff
@@ -26,50 +26,43 @@
 
 #ifdef LMMS_HAVE_LV2
 
+#include <QDebug>
+#include <QDir>
+#include <QElapsedTimer>
+#include <QLibrary>
 #include <algorithm>
 #include <cstdlib>
 #include <cstring>
 #include <lilv/lilv.h>
 #include <lv2.h>
 #include <lv2/lv2plug.in/ns/ext/options/options.h>
-#include <QDebug>
-#include <QDir>
-#include <QLibrary>
-#include <QElapsedTimer>
 
 #include "ConfigManager.h"
 #include "Engine.h"
-#include "Plugin.h"
-#include "PluginFactory.h"
 #include "Lv2ControlBase.h"
 #include "Lv2Options.h"
+#include "Plugin.h"
+#include "PluginFactory.h"
 #include "PluginIssue.h"
 
-
-
-
-const std::set<const char*, Lv2Manager::CmpStr> Lv2Manager::pluginBlacklist =
-{
-	// github.com/calf-studio-gear/calf, #278
-	"http://calf.sourceforge.net/plugins/Analyzer",
-	"http://calf.sourceforge.net/plugins/BassEnhancer",
-	"http://calf.sourceforge.net/plugins/CompensationDelay",
-	"http://calf.sourceforge.net/plugins/Crusher",
-	"http://calf.sourceforge.net/plugins/Exciter",
-	"http://calf.sourceforge.net/plugins/Saturator",
-	"http://calf.sourceforge.net/plugins/StereoTools",
-	"http://calf.sourceforge.net/plugins/TapeSimulator",
-	"http://calf.sourceforge.net/plugins/TransientDesigner",
-	"http://calf.sourceforge.net/plugins/Vinyl"
-};
-
-
-
+const std::set<const char *, Lv2Manager::CmpStr> Lv2Manager::pluginBlacklist =
+	{
```

Comment:
This indentation looks bad to me, though it's due to the initialization: the opening bracket is still part of the whole initialization, so it continues the first line, and must be indented.

Reply:
So I think this is OK.